borg/paperkey.html

```diff
@@ -2217,7 +2220,7 @@
     <div id="printout">
         <div id="title" contenteditable>BorgBackup Printable Key Backup</div>
         <div contenteditable>To restore either scan the QR code below, decode it and import it using
-<pre>borg key-import /path/to/repo scannedfile</pre>
+<pre>borg key import /path/to/repo scannedfile</pre>
```

Comment:
hmm, what is the format (mimetype) of scannedfile?

"scannedfile" sounds a bit like .tiff or other bitmap format, but I guess it needs to be text? or binary?